onyo/lib/command_utils.py

```diff
@@ -68,8 +68,8 @@ def sanitize_keys(k: Optional[list[str]],
     Remove duplicates from k while preserving key order and return default
     (pseudo) keys if k is empty
     """
-    seen = set()
-    return [x for x in k if not (x in seen or seen.add(x))] if k else defaults
+    from .utils import deduplicate
+    return deduplicate(k) if k else defaults
```

Comment:
I know `get` is unconverted, so these comments are more for later reference:

- the name `sanitize_keys()` is likely a leftover from when this function did more; to me the name implies that it does more
- its location (and name) implies that it's generic, but it has one consumer (`get`), and I struggle to think of what else would need this functionality

This function has one consumer, and is a one-liner. Perhaps it is best folded directly into `get` (when that effort happens).

Reply:
Agree. That should and will be part of addressing `get`, though.